src/types/nft.ts

```diff
@@ -116,3 +122,7 @@ export const getTrait = (nft: NFT, type: string) => {
   }
   return traitAttribute.value;
 };
+
+export const fieldExtractors: any = {
+  'metadata.name': (nft: NFT) => { return nft.metadata.name }
```

Comment:
lets actually use getTrait(nft, 'Song Title') for this. i notice that metadata.name has an extra - 01 at the end since it's with the specific nft print number in the name rather than just the song title